lib/spack/spack/binary_distribution.py

```diff
@@ -1388,40 +1386,18 @@ def extract_tarball(spec, filename, allow_root=False, unsigned=False,
     buildinfo = spec_dict.get('buildinfo', {})
     old_relative_prefix = buildinfo.get('relative_prefix', new_relative_prefix)
     rel = buildinfo.get('relative_rpaths')
-    # if the original relative prefix and new relative prefix differ the
-    # directory layout has changed and the  buildcache cannot be installed
-    # if it was created with relative rpaths
```

Comment:
Was this comment referring to the commented out code below?

Reply:
No this was left over from before I added code to allow installation to a different layout.